### ISSUE TITLE
Multi-Tenant Proxy HPA - missing cpu requests

### ISSUE THREAD

**Issue by jkrusy** [giantswarm/loki-app#207]
Hello,

the HPA for the multi-tenant-proxy is not working because it can not get the CPU requests
> the HPA was unable to compute the replica count: failed to get cpu utilization: missing request for cpu

Two of the three containers in the pod do not have a cpu request set and this can not be set in the chart:
https://github.com/giantswarm/loki-app/blob/master/helm/loki/templates/multi-tenant-proxy/multi-tenant-proxy.yaml#L96
https://github.com/giantswarm/loki-app/blob/master/helm/loki/templates/multi-tenant-proxy/multi-tenant-proxy.yaml#L114

When these resources are set, the HPA is working again.

**Comment by TheoBrigitte** [comment]
Hello, thanks for reporting this issue. I will make sure we address this.

**Comment by TheoBrigitte** [comment]
Here is the full describe of the `loki-multi-tenant-proxy` hpa
```
k -n=loki describe hpa loki-multi-tenant-proxy
Warning: autoscaling/v2beta2 HorizontalPodAutoscaler is deprecated in v1.23+, unavailable in v1.26+; use autoscaling/v2 HorizontalPodAutoscaler
Name:                                                  loki-multi-tenant-proxy
Namespace:                                             loki
Labels:                                                app=loki-multi-tenant-proxy
                                                       app.kubernetes.io/instance=loki
                                                       app.kubernetes.io/managed-by=Helm
                                                       app.kubernetes.io/name=loki
                                                       app.kubernetes.io/version=2.8.4
                                                       application.giantswarm.io/team=atlas
                                                       giantswarm.io/service-type=managed
                                                       helm.sh/chart=loki-0.11.1
Annotations:                                           meta.helm.sh/release-name: loki
                                                       meta.helm.sh/release-namespace: loki
CreationTimestamp:                                     Mon, 28 Aug 2023 12:48:30 +0200
Reference:                                             Deployment/loki-multi-tenant-proxy
Metrics:                                               ( current / target )
  resource cpu on pods  (as a percentage of request):  <unknown> / 60%
Min replicas:                                          2
Max replicas:                                          4
Deployment pods:                                       2 current / 2 desired
Conditions:
  Type           Status  Reason                   Message
  ----           ------  ------                   -------
  AbleToScale    True    SucceededGetScale        the HPA controller was able to get the target's current scale
  ScalingActive  False   FailedGetResourceMetric  the HPA was unable to compute the replica count: failed to get cpu utilization: missing request for cpu
Events:
  Type     Reason                   Age                     From                       Message
  ----     ------                   ----                    ----                       -------
  Warning  FailedGetResourceMetric  4s (x45957 over 7d23h)  horizontal-pod-autoscaler  failed to get cpu utilization: missing request for cpu
```

**Comment by hervenicol** [comment]
Fix is included in loki-app v0.12.0.
